aexpect/remote.py

```diff
@@ -353,6 +368,8 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
             cmd_status_output() and friends).
     :param bind_ip: ssh through specific interface on
                     client(specify interface ip)
+    :param preferred_authenticaton: The preferred authentication of SSH connection
+    :param user_known_hosts_file: one or more files to use for the user host key database
```

Comment:
I would either have both parameters described with capital letters or both with small letters but just commenting here.

Reply:
I updated it to ':param user_known_hosts_file: Given value of UserKnownHostsFile in ssh', is it ok?

Reply:
Sure, either one works at least for a minor consistency - I checked your change and it is fine.